src/clang.rs

```diff
@@ -645,6 +645,24 @@ impl Cursor {
         self.has_attr("warn_unused_result", Some(CXCursor_WarnUnusedResultAttr))
     }
 
+    pub fn has_no_return_attr(&self) -> bool {
```

Comment:
We've found performance issues with these kinds of calls in the past and we have a `enable_function_attribute_detection` option already. Can we use that?

Reply:
Sure. I'll add an extra branch checking that `enable_function_attribute_detection` is enabled and use that when checking for this attribute.